plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -273,6 +267,8 @@ class CodeModernizerSession(
             ) { shouldStop.get() }
         } catch (e: Exception) {
             val errorMessage = "Unexpected error when uploading artifact to S3: ${e.localizedMessage}"
+            LOG.error { errorMessage }
```

Comment:
Caller `createModernizationJob` seems to also log this error message. Will this be redundant?

Reply:
See above, I removed it from there and put it here instead.